src/context/online.rs

```diff
@@ -136,11 +153,13 @@ impl AudioContext {
 
         let (sender, _receiver) = crossbeam_channel::unbounded();
         let frames_played = Arc::new(AtomicU64::new(0));
+        let output_latency = Arc::new(AtomicF64::new(0.));
 
         let base = ConcreteBaseAudioContext::new(
             sample_rate,
             number_of_channels,
             frames_played,
+            output_latency,
```

Comment:
This attribute should actually be inside the AudioContext, not the ConcreteBaseAudioContext
This means we can leave it out of the offline.rs file too!